cfgov/ask_cfpb/models/pages.py

```diff
@@ -288,6 +288,18 @@ class TagResultsPage(RoutablePageMixin, AnswerResultsPage):
 
     objects = CFGOVPageManager()
 
+    def set_language(self):
```

Comment:
Would it make sense to add this to `CFGOVPage` directly? Or is it only needed on a couple page types?

Reply:
CFGOVPage already [has activization](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py#L224-L225), but it's in the `serve` method, which routable pages do not call. That's why they need their own.

The current routables are probably going away soon, so I'm not too concerned about repeated code at this point. I'm looking for a good way to add translations to all routables. Unfortunately the mixin isn't the place.